### PR TITLE
SQL Server no longer add Administratos

### PR DESCRIPTION
**Pull request by taroarai**
SQL Server no longer add `BUILTIN\Administrators` group since SQL Server 2005, so could you remove this as SQL Server 2005 is no longer supported, or add what I add as it is until SQL Server 2005.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by gewarren** [comment]
@mashamsft Could you verify this?

File: docs/framework/data/adonet/sql/authentication-in-sql-server.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ SQL Server supports two authentication modes, Windows authentication mode and mi
...
+> SQL Server installs with a SQL Server login named `sa` (an abbreviation of "system administrator"). Assign a strong password to the `sa` login and do not use the `sa` login in your application. The `sa` login maps to the `sysadmin` fixed server role, which has irrevocable administrative credentials on the whole server. There are no limits to the potential damage if an attacker gains access as a system administrator. All members of the Windows `BUILTIN\Administrators` group (the local administrator's group) are members of the `sysadmin` role by default (Until SQL Server 2005), but can be removed from that role.
```
Comment (gewarren) [inline docs/framework/data/adonet/sql/authentication-in-sql-server.md:66]:
```suggestion
> SQL Server installs with a SQL Server login named `sa` (an abbreviation of "system administrator"). Assign a strong password to the `sa` login and do not use the `sa` login in your application. The `sa` login maps to the `sysadmin` fixed server role, which has irrevocable administrative credentials on the whole server. There are no limits to the potential damage if an attacker gains access as a system administrator.
```

**Review by gewarren — DISMISSED** [review]
@taroarai Thanks for noticing this. We decided the sentence should just be removed completely.

**Review by adegeo — APPROVED** [review]
@gewarren I went ahead and committed your suggestion as it's the appropriate fix.

**Pull request merged**